custom/enikshay/management/commands/get_users_for_bets.py

```diff
@@ -53,26 +107,24 @@ def handle(self, domain, **options):
                     writer.writerow([username, user_id])
 
     def add_user(self, user, writer):
-        if (user.user_data.get('usertype', None) not in ['pcp', 'pcc-chemist', 'plc', 'pac']
-                or user.user_data.get('user_level', None) != 'real'):
-            return
 
-        location_id = user.user_location_id
-        location = self.locations_by_id.get(location_id, None) if location_id else None
-        if not location:
-            self.locationless_users.append((user.username, user._id))
+        def _is_relevant_location(location):
+            return (location.metadata.get('is_test') != "yes"
+                    and location.location_type.code in BETSUserRepeater.location_types_to_forward)
+
+        user_locations = [self.locations_by_id.get(loc_id) for loc_id in user.get_location_ids(self.domain)]
+        if not (user.user_data.get('user_level', None) == 'real'
+                and any(_is_relevant_location(loc) for loc in user_locations)):
             return
 
-        writer.writerow([
-            user._id,
-            user.raw_username,
-            user.first_name,
-            user.last_name,
-            location_id,
-            location.name,
-            user.phone_number,
-            user.email,
-            user.language,
-        ] + [
-            user.user_data.get(field.slug, '') for field in self.data_fields
-        ])
+        user_data = BETSUserPayloadGenerator.serialize(self.domain, user)
+
+        def get_field(field):
+            if field == 'phone_numbers':
+                return ''
+            elif '.' in field:
+                obj, key = field.split('.')
+                return user_data[obj].get(key, '')
+            return user_data.get(field, '')
```

Comment:
won't `migrationCrtdRecordId` always end up as `''` here? I thought it needed to be set to the UATBC id of the agency.